enhancements/node/cri-o-operator.md

```diff
@@ -0,0 +1,307 @@
+---
+title: cri-o-operator
+authors:
+  - "@saschagrunert"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-06-22
+last-updated: 2021-06-22
+status: provisional
+see-also: {}
+replaces: {}
+superseded-by: {}
+---
+
+# CRI-O operator
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The overall goal of creating this new operator in OpenShift is the encapsulation
+of CRI-O related configurations, metrics and alerts within a single domain and
+namespace. This provides on one hand a higher flexibility for the Node team,
```

Comment:
the summary here implies something the rest of this doc does not go into: what are we going to do with ContainerRuntimeConfig handling? is that being moved here, or is this only cri-o metrics related?

Reply:
Yeah, made it more clear to handle kubelet and runtime related configurations too if an API is available.